app/controllers/ExclusionListController.scala

```diff
@@ -93,10 +93,10 @@ class ExclusionListController @Inject()(
       if (exclusionsAllowed) {
         val iabdTypeValue = uriInformation.iabdValueURLDeMapper(iabdType)
         val staticDataRequest = for {
-          year                                              <- validateRequest(isCurrentTaxYear, iabdType)
-          currentYearList: (Map[String, String], List[Bik]) <- bikListService.currentYearList
-          nextYearList: (Map[String, String], List[Bik])    <- bikListService.nextYearList
-          currentYearEIL: List[EiLPerson]                   <- eiLListService.currentYearEiL(iabdTypeValue, year)
+          year <- validateRequest(isCurrentTaxYear, iabdType)
+          //currentYearList: (Map[String, String], List[Bik]) <- bikListService.currentYearList
```

Comment:
Commented out code.

Reply:
forgot to delete the line it's unused, just commented out to run the tests to verify